### DISCUSSION TITLE
使用率超过的提示标题修改

### DISCUSSION THREAD

**Discussion by Wangsc1** [zdz/ServerStatus-Rust#15] [Q&A] [answered]
![01D878FA-27D4-4BE6-A703-C663FCBD8A86](https://user-images.githubusercontent.com/26079238/161422900-89dd35b3-c55a-4195-a5f7-8dfae4f0ffad.png)
大佬，这个标题在哪能修改，和上下线的提示标题不是同一个

**Comment by zdz** [comment]
目前在代码里，没办法修改
https://github.com/zdz/ServerStatus-Rust/blob/9e08de85be5f109c40cec590bf8d6c8d69d44d5a/server/src/notifier/tgbot.rs#L106

**Reply by Wangsc1** [reply]
这个tgbot.rs在服务器哪个位置呢，我来手动改下

**Comment by zdz** [comment]
@Wangsc1 后面在这里跟进

**Comment by zdz** [comment]
点上面链接,跳过去就看到了,要重新编译一下

**Reply by Wangsc1** [reply]
不会搞😂，算了就这样吧，打扰了🙏

**Comment by zdz** [comment, marked as answer]
https://github.com/zdz/ServerStatus-Rust/releases/tag/v1.3.3

**Reply by Wangsc1** [reply]
感谢大佬🙏
